templates/components/common/share.html

```diff
@@ -61,7 +61,7 @@
             {{/each}}
         </ul>
         <script type="text/javascript"
-                src="//s7.addthis.com/js/300/addthis_widget.js#pubid=ra-4e94ed470ee51e32"></script>
+                src="//s7.addthis.com/js/300/addthis_widget.js#pubid=ra-4e94ed470ee51e32" defer></script>
```

Comment:
I don't think it's this simple, because the script just below depends on this script loading first.